hwangsae/transmuxer.h

```diff
@@ -25,6 +25,8 @@
 
 #include <glib-object.h>
 
+#include "types.h"
+
```

Comment:
This change seems not to be necessary for header. Shouldn't we move `types.h` to `.c` file?

Reply:
I added it to allow to reference HWANGSAE_TRANSMUXER_ERROR_OVERLAP from application without including, similar approach to the one used in `recorder.h`.

Do you prefer `types.h` be included in application?

Reply:
I think it's better to keep `.h` files simple. When I moved this statement to `.c`, it was built successfully. 

And if you want to keep `types.h` for public header, it should be `<types.h>` for consistency.

Reply:
Thanks for your time and review. You are right, as `types.h` is in `hwangsae.h` it is not necessary here as I initially supposed.